cmd/minishift/cmd/config/validations.go

```diff
@@ -41,7 +41,7 @@ func IsValidDriver(string, driver string) error {
 }
 
 func RequiresRestartMsg(name string, value string) error {
-	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+	api := libmachine.NewClient(constants.GetMinipath(), constants.MakeMiniPath("certs"))
```

Comment:
Will look in to this and get back. I would be interested to reduce the code change.